### DISCUSSION TITLE
Several drones and only one ground station

### DISCUSSION THREAD

**Discussion by InanixFR** [svpcom/wfb-ng#204] [Ideas]
Hello,

I would like to use Wifibroadcast on several drones (6) at the same time, but using only one ground station server (for sure with 6 Wifi cards!). 
What I understand is when I launch `sudo wfb-server gs wlanX` it goes to check configuration in file `/etc/wifibroadcast.cfg`
On enhancement could be to add a parameter to wfb-server to set the configuration file path.

But for your opinion, is it possible? is it enough?

Thanks

**Comment by svpcom** [comment]
Yes, it is enough. You need to use non-overlapping wifi channels. Also you can use custom encryption keys for each instance to avoid command interference between drones if two drones are accidentally set on one frequency channel. I've added support for custom config path in master branch. You can set `WIFIBROADCAST_CFG` environment variable to custom config for each instance

**Reply by InanixFR** [reply]
Great!!! So quickly !!!

Thanks a lot
